Looms.PoS/Looms.PoS.Domain/Enums/OrderStatus.cs

```diff
@@ -0,0 +1,15 @@
+using System.Runtime.Serialization;
+
+namespace Looms.PoS.Domain.Enums;
+
+public enum OrderStatus
+{
+    [EnumMember(Value = "Pending")]
```

Comment:
no need for enum member when it is exactly the same

Reply:
removed